pkg/graphql/request.go

```diff
@@ -109,6 +110,10 @@ func (r *Request) IsNormalized() bool {
 	return r.isNormalized
 }
 
+func (r *Request) IsValidated() bool {
+	return r.isValidated
```

Comment:
`isNormalized` is set to `true` here: https://github.com/TykTechnologies/graphql-go-tools/blob/92655e8c2e963252c0e5a742925d0bc0d536e763/pkg/graphql/normalization.go#L43

We need the same thing for `isValidated`. There is a file called `validation.go`. We should add the same logic here: https://github.com/TykTechnologies/graphql-go-tools/blob/master/pkg/graphql/validation.go

Reply:
`ValidateForSchema` is called by GW and this should work but there are a few more functions in `validate.go` file. Do we need to add the same logic to `ValidateFieldRestrictions` and `ValidateRestrictedFields` methods? Possibly not but we need to investigate that before merging. 

Also, adding simple unit tests for `IsNormalized` and `IsValidated` methods is a good idea.